src/commonTest/kotlin/serialize/TestRoutes.kt

```diff
@@ -0,0 +1,22 @@
+package serialize
+
+import com.algolia.search.serialize.*
+import shouldEqual
+import kotlin.test.Test
+
+
+internal class TestRoutes {
```

Comment:
Not sure how useful that is, considering you took those strings from the same place than when writing the keys :man_shrugging:

Reply:
I mean if we start to write tests to confirm we copy-pasted correctly, it will never end :upside_down_face: